examples/04_visualize-ner-extra-options.py

```diff
@@ -0,0 +1,35 @@
+"""
+Example of using extra_options for visualize_ner.
+"""
+import spacy
+
+import spacy_streamlit
+
+nlp = spacy.blank("en")
+text = "But Google is starting from behind."
+doc = nlp.make_doc(text)
+ent = doc.char_span(4, 10, label="ORG", kb_id="Q95")
+doc.ents = [ent]
+
+spacy_streamlit.visualize_ner(
+    doc,
+    labels=["ORG"],
+    show_table=False,
+    title="Custom Colors NER Visualization",
+    colors={"ORG": "#EEE"},
```

Comment:
We could add in the `colors` to the `displacy_options` to show good practice going forward?

Reply:
Done.